### ISSUE TITLE
CSS Modules generateScopedName not working

### ISSUE THREAD

**Issue by brielov** [parcel-bundler/parcel#2988]
I'm not able to override the default naming for css module class names. No matter what I put in generateScopedName, it'll always output the original convention.

```json
{
  "modules": true,
  "plugins": {
    "autoprefixer": {
      "grid": true
    },
    "postcss-modules": {
      "generateScopedName": "[hash:base64:5]"
    }
  }
}
```

**Comment by mischnic** [comment]
This was already fixed a few days ago in #2922. This will be fixed in the next release.

**Comment by brielov** [comment]
Oh, my bad... thank you for the quick reply!
